docs/languages/polyglot.md

```diff
@@ -0,0 +1,100 @@
+# Polyglot Notebooks
+
+## What is Polyglot Programming?
+
+Polyglot programming is the practice of using multiple programming languages to leverage the strength of each language for different tasks.
+
+## What are Notebooks?
+Notebooks are interactive files that allow the mixing of executable code, visualizations, equations and narrative text. Notebooks are composed of code cells that make it easy to quickly iterate on code. Created and popularized by the open-source project [Jupyter](https://jupyter.org/), they have become the de facto tool for data science and a great resource for teaching or learning a new programming language and quick prototyping.
+
+
+
+## Polyglot Notebooks
+
+When polyglot programming meets notebooks, you get Polyglot Notebooks! As opposed to traditional notebooks that are typically used with Python, the [Polyglot Notebooks Extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode) allows you to use multiple programming languages natively all in the same notebook! No more needing
```

Comment:
nit: Seems like an incomplete sentence...

Reply:
Oh oops I think I moved this info elsewhere

Reply:
I just finished the thought and added it back